tests/unit/validation/test_validation_functions.py

```diff
@@ -33,7 +34,7 @@
     validate_values,
 )
 
-NaN = power_grid_meta_data["input"]["node"].nans["id"]
+NaN = power_grid_meta_data[DataType.input][ComponentType.node].nans["id"]
```

Comment:
This line really beats the purpose of this PR...

Reply:
@Jerry-Jinfeng-Guo why? `power_grid_meta_data` takes both `Literal`s and `DataType`s / `ComponenType`s